### DISCUSSION TITLE
Home Button

### DISCUSSION THREAD

**Discussion by bpmct** [coder/code-server#2761] [General] [answered]
I have noticed this new home button in code-server 3.9.0. Is this something related to a new version of VS Code, or was it added in code-server? To me, it is a bit confusing, and looks like I have a notification.

<img width="84" alt="Screen Shot 2021-02-24 at 9 36 23 AM" src="https://user-images.githubusercontent.com/22407953/109017440-2d706280-7685-11eb-9d00-9a55c2db1a91.png">

**Comment by jsjoeio** [comment, marked as answer]
Before, I think there were to Home options. 

1. the icon you screenshotted 
2. the "Go Home" in the hamburger menu

I think when we upgraded to VSCode 1.52.1, they removed it from the hamburger menu. [See comment](https://github.com/cdr/code-server/pull/2543#issuecomment-774246668).

> To me, it is a bit confusing, and looks like I have a notification

Yeah, I don't love it either! We could either redesign it or hide it? I'd be open to either!

**Reply by code-asher** [reply]
Users can disable it by right-clicking on it or the sidebar I think but I don't care for it much either. :stuck_out_tongue: Maybe we default the hidden option to true.

**Reply by jsjoeio** [reply]
Oh right! I forgot about that! Lol doesn't matter to me either, @bpmct feel free to submit a PR if you want 😛

**Reply by code-asher** [reply]
Oh another option would be to remove our default and make it so it'll only show when you explicitly pass `--home`. That might be the best option. It's honestly kind of an unsightly button IMO.

**Reply by code-asher** [reply]
And then we wouldn't have to implement https://github.com/cdr/code-server/issues/2742 haha since the only reason we need that right now is because of the home button which we could just omit for enterprise.

**Reply by jsjoeio** [reply]
Genius! Well we could modify #2742 to be "remove our default and make it so it'll only show when you explicitly pass --home"

**Reply by nooobcoder** [reply]
@bpmct this thing worried me too
It was so confusing then disabled it.

**Reply by jsjoeio** [reply]
Update: this will be removed when https://github.com/cdr/code-server/pull/2862 is merged
